platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -173,10 +173,10 @@ index e4b1b0f1c..36cd6aacf 100644
  import android.util.Log;
  
 -import com.android.vending.expansion.downloader.R;
-+// -- GODOT start --
++// -- BLAZIUM start --
 +//import com.android.vending.expansion.downloader.R;
```

Comment:
The line 177-178 maybe from a different branch, please validate this is required for rebranding

Reply:
checked, the same!, this is some sort of patch note i think